src/utils/image-overlay.js

```diff
@@ -8,7 +8,13 @@ import loadImage from "easy-image-loader";
 // (3) rejects the promise if it takes more than 5 seconds for the image to load
 export default async function imageOverlay(url, bounds, crossOrigin, options) {
   const timeout = 5 * 1000; // 5 seconds
-  const img = await loadImage(url, { crossOrigin, timeout });
+  let img = null
+  try {
+    img = await loadImage(url, { crossOrigin, timeout });
+  }
+  catch {
+    return null
```

Comment:
I like how we are returning null now, instead of throwing an error.  Could we take this further and wrap the whole imageOverlay function in a `try ... catch { return null; }`.  That way the user can be assured that if there is an error whether in calling loadImage or L.imageOverlay, the function returns null.  I find it a little confusing to sometimes return `null` or sometimes return a rejected promise depending on the type of error.  Here's a possible update to the comments:
```js
// practically identical to L.imageOverlay
// with the following exceptions:
// (1) it is async and returns a promise
// (2) if there is any error, the returned promise resolves to null 
```